### PR TITLE
Add notes from 2023-05-19

### PR DESCRIPTION
**Pull request by tanho63**

### CODE REVIEW

File: 02_tidy-evaluation-metaprogramming.Rmd [hunk 1]
```diff
@@ -317,6 +316,63 @@ my_pivot_longer <- function(data, ...) {
 mtcars %>% my_pivot_longer(cyl, am = am * 100)
 ```
 
+### Double Evaluation {-}
```
Comment (tanho63) [inline 02_tidy-evaluation-metaprogramming.Rmd]: I forget exactly how bookdown works, is it ## for a new page altogether?

Comment (jonthegeek) [reply 02_tidy-evaluation-metaprogramming.Rmd]: Yup!

Comment (tanho63) [reply 02_tidy-evaluation-metaprogramming.Rmd]:
```suggestion
## Guide: Double Evaluation {-}
```

File: 02_tidy-evaluation-metaprogramming.Rmd [hunk 2]
```diff
@@ -317,6 +316,63 @@ my_pivot_longer <- function(data, ...) {
...
+create an intermediate variable
+
+(aside: lol transmute again?)
+
+##### Symbols / glue strings {-}
+Glue strings and other expression/symbol evaluation doesn't "suffer" from this problem
+
+- "not suffering" meaning it doesn't execute code and instead translates the code to text
+
+### Injection Operators Out Of Context {-}
```
Comment (tanho63) [inline 02_tidy-evaluation-metaprogramming.Rmd:344]:
```suggestion
## Tidy Eval: Notes {-}
### Injection Operators Out Of Context {-}
```

File: 02_tidy-evaluation-metaprogramming.Rmd [hunk 3]
```diff
@@ -317,6 +316,63 @@ my_pivot_longer <- function(data, ...) {
...
+If you use `{{` on regular objects (and not on arguments), it captures the value 
+of the object. This is related to how enquo works. 
+
+### Function references {-}
+
+https://rlang.r-lib.org/reference/eval_tidy.html - underlying function that powers data masking
+
+https://rlang.r-lib.org/reference/dot-data.html - documentation for the pronouns
+
+https://rlang.r-lib.org/reference/as_data_mask.html - underlying functions inside of eval_tidy
```
Comment (tanho63) [inline 02_tidy-evaluation-metaprogramming.Rmd]:
```suggestion
### Function references {-}

https://rlang.r-lib.org/reference/eval_tidy.html - underlying function that powers data masking

https://rlang.r-lib.org/reference/as_data_mask.html - underlying functions inside of eval_tidy
```

**Review by jonthegeek — APPROVED** [review]
Thanks!

**Pull request merged**